| Name | Department/Program | Experties/Interests |GitHub ID | 
| ------------- | ------------- | ------------- | ------------- |
| example: Santina Lin | Bioinformatics  | Machine learning and data mining  |  @santina |
|  |   |  |
|  |   |  |
|  |   |  |
|  |   |  |

Team name: 

One paragraph on the basic idea of the project: 
Proposal Draft

"What is the relationship between DNA methylation and neuronal proportion in the cortex versus cerebellum of middle aged adults?"

DNA methylation (DNAm), or covalent attachment of a methyl group to the 5’ position of cytosine bases located in
CpG dinucleotides, plays a crucial role in maintaining patterns of gene expression during human development and aging. 
Along with other epigenetic modifications, DNAm is sensitive to environmental influences and can change over an individual’s
lifespan. Thus, understanding the human methylome is important for determining both biomarkers for - and direct pathways implicating - health and disease [1]. Aberrant DNAm patterns have been correlated with common neurodegenerative disorders, including Alzheimer’s Disease and Parkinson’s Disease [2], as well as with mental disorders such as schizophrenia [3,4]. However, much of the literature on either the diseased or healthy brain methylome fails to separate DNAm data by cell type composition - a major driver of DNAm variability - or by brain region [5, 6]. We aim to provide a baseline of variation in the normal human methylome across various brain regions and according to proportions of neurons versus glia. We will use a publically available dataset of n=122 healthy individuals aged 40-105 [7], consisting of reads obtained using the Illumina HumanMethylation450 BeadChip array platform. We will process the reads and quantify methylation levels, determine the proportions of neurons in the prefrontal cortex, entorhinal cortex, superior temporal gyrus and the cerebellum with a cell type prediction algorithm, and correlate the neuronal proportion with the amount of methylation in these regions [8]. This analysis will inform on (1) the difference in methylation levels between cortex and cerebellum and within cortical subregions and (2) the relationship within neuronal proportion and methylation levels in all regions analyzed. Our findings will be validated using an independent dataset: the Allen Brain Atlas’ Human Developing Brain cohort of n=16 healthy individuals aged 4 months - 37 years (http://brain-map.org/). Ultimately, the project will shed light on the importance of variability in methylation between brain regions, providing a guideline to analyze existing brain epigenetic literature and to plan robust experimental design moving forward.
